Makefile

```diff
@@ -806,4 +809,12 @@ act_list: check_act ## List all github actions that can be executed locally with
 act_reviewdog: check_act check_gh ## Run the reviewdog workflow locally like so: `GITHUB_TOKEN=$(gh auth token) make act_reviewdog`
 	$(eval CONTAINER_ARCH := $(shell make -s detect_arch))
 	@echo "Detected architecture: $(CONTAINER_ARCH)"
-	act -v -s GITHUB_TOKEN=$(GITHUB_TOKEN) -W .github/workflows/reviewdog.yml --container-architecture $(CONTAINER_ARCH)
\ No newline at end of file
+	act -v -s GITHUB_TOKEN=$(GITHUB_TOKEN) -W .github/workflows/reviewdog.yml --container-architecture $(CONTAINER_ARCH)
+
+#############################
+### Grove Gateway Helpers ###
+#############################
+
+.PHONY: grove_eth_balance
+grove_eth_balance: # Must have GROVE_PORTAL_APP_ID environment variable set
```

Comment:
1. Look at all the other targets. This isn't just a comment, it's also documentation
2. Add details on what it does.